docs/concepts/automations.md

```diff
@@ -391,7 +391,163 @@ To detect when the average lateness of your Kubernetes workloads (running on a w
 
 ### Composite triggers
 
-[more information coming soon]
+To express scenarios involving several kinds of events and metrics, you can use Prefect's `compound` and `sequence` triggers.  These higher-order triggers are composed from combinations of underlying `event` and `metric` triggers, and will fire when some combination of those underlying triggers do.
+
+For example, if you want to run a deployment only after 3 different flows in your workspace have written their results to an S3 bucket, combine them with a 'compound' trigger:
+
+```json
+{
+  "type": "compound",
+  "require": "all",
+  "within": 3600,
+  "triggers": [
+    {
+      "type": "event",
+      "posture": "Reactive",
+      "expect": ["prefect.block.remote-file-system.write_path.called"],
+      "match_related": {
+        "prefect.resource.name": "daily-customer-export",
+        "prefect.resource.role": "flow"
+      }
+    },
+    {
+      "type": "event",
+      "posture": "Reactive",
+      "expect": ["prefect.block.remote-file-system.write_path.called"],
+      "match_related": {
+        "prefect.resource.name": "daily-revenue-export",
+        "prefect.resource.role": "flow"
+      }
+    },
+    {
+      "type": "event",
+      "posture": "Reactive",
+      "expect": ["prefect.block.remote-file-system.write_path.called"],
+      "match_related": {
+        "prefect.resource.name": "daily-expenses-export",
+        "prefect.resource.role": "flow"
+      }
+    }
+  ]
+}
+```
+
+This trigger will fire once it sees at least one of each of the underlying events happen, then it will reset it's state waiting for the next time these three events all happen again.  The order of the events won't matter, just that they all happen within an hour of one another.
```

Comment:
```suggestion
This compound trigger will fire once it sees at least one of each of the underlying events happen, then it will reset its state waiting for the next time these 3 events all happen again.  The order of the events won't matter, just that they all happen within an hour of one another.
```

Using the actual number makes this easier to digest imo

> within an hour of one another
I'm not totally sure this is clear. That makes it sound to me like A could happen at hour 0 and B could happen at hour 1 and C could happen at hour 2 and it would trigger.

Reply:
Oh great point, that needs to be restated